tests/test_skyModel.py

```diff
@@ -3,22 +3,34 @@
 """
 import os
 import copy
-import time
 import unittest
-from collections import namedtuple
-try:
-    import configparser
-except ImportError:
-    # python 2 backwards-compatibility
-    import ConfigParser as configparser
+import pickle
+import configparser
 import numpy as np
 import numpy.random as random
 import galsim
-import lsst.sims.skybrightness as skybrightness
 from lsst.sims.GalSimInterface import make_galsim_detector, LSSTCameraWrapper
-from lsst.sims.photUtils import BandpassDict
 import desc.imsim
 
+
+class DC2SkyModel:
+    """
+    Minimal SkyModel class to provide the sky SED evaluated at
+    skyModel.setRaDecMjd(0., 90., 58000, azAlt=True, degrees=True)
+    using data from the DC2Production sims_skybrightness_data tag.
```

Comment:
The bandpasses are read in separately.  See line 174 below, where the DC2Production versions are read in rather than relying on whatever `throughputs` package is being used.  I think the sky model data in `sims_skybrightness_data` must be independent of the `throughputs`.   This code is just to guard against the sky model data changing and from which the sky SED is obtained.  The sky SED shape is used even when `magNorm` is set below, so changes in `sims_skybrightness_data` would affect these tests.

Reply:
Ah! I missed that the pickle files were different went I went over it. I was actually confused by that.. OK, got it. Thanks.